src/app/backend/apihandler.go

```diff
@@ -18,17 +18,39 @@ import (
 	"log"
 	"net/http"
 	"strconv"
+	"strings"
 
 	restful "github.com/emicklei/go-restful"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 )
 
+func WsLogger() restful.FilterFunction {
+	return func(req *restful.Request, resp *restful.Response, chain *restful.FilterChain) {
+		reqBody := make([]byte, req.Request.ContentLength)
+		req.Request.Body.Read(reqBody)
+		remoteAddr := strings.Split(req.Request.RemoteAddr, ":")[0]
+
+		log.Printf("Incoming %s %s %s request from %s with %d content length:\n%v",
+			req.Request.Proto, req.Request.Method, req.Request.URL.RequestURI(), remoteAddr,
+			req.Request.ContentLength, reqBody,
+		)
+
+		chain.ProcessFilter(req, resp)
+		respBody := "[]" // TODO(maciaszczykm): Get response content.
+
+		log.Printf("Outcoming response to %s with %d status code and %d content length:\n%v",
+			remoteAddr, resp.StatusCode(), resp.ContentLength(), respBody,
```

Comment:
Okay, I'll remove. I was using it before displaying content to check if there is any.